src/pymatgen/io/lobster/outputs.py

```diff
@@ -855,9 +912,16 @@ def get_structure_with_charges(self, structure_filename: PathLike) -> Structure:
             Structure Object with Mulliken and Loewdin charges as site properties.
         """
         struct = Structure.from_file(structure_filename)
-        mulliken = self.mulliken
-        loewdin = self.loewdin
-        site_properties = {"Mulliken Charges": mulliken, "Loewdin Charges": loewdin}
+        site_properties = {}
+        if "LCFO" not in Path(self._filename).name:
```

Comment:
Not sure this is really a safe check. Also, not sure if fragment orbital charges can ever be assigned to a specific site. I would throw an error.

Reply:
Have updated the method to raise a ValueError now for LCFO files.